util/src/main/scala/org/broadinstitute/dsde/rawls/util/Retry.scala

```diff
@@ -15,13 +15,15 @@ trait Retry {
   this: LazyLogging =>
   val system: ActorSystem
 
+  type Predicate[A] = A => Boolean
```

Comment:
I like this.  Would you mind bringing `def always` up here and defining it in terms of `Predicate`?